src/main/java/com/github/technus/tectech/thing/metaTileEntity/multi/GT_MetaTileEntity_EM_EyeOfHarmony.java

```diff
@@ -1111,13 +1146,40 @@ public CheckRecipeResult processRecipe(EyeOfHarmonyRecipe recipeObject) {
             }
         }
 
+        astralArrayAmount = 0;
+
+        for (int i = 0; i < mInputBusses.get(0).getSizeInventory(); i++) {
```

Comment:
Might be better to use an enhanced for loop here if possible? I think you can get the entire inventory with one of those Compacted methods